test/contrib/test_gae_manager.py

```diff
@@ -177,8 +177,24 @@ def test_default_method_whitelist_retried(self):
             retries=retry)
         self.assertEqual(resp.status, 200)
 
+    def test_retry_return_in_response(self):
+        headers = {'test-name': 'test_retry_return_in_response'}
+        retry = Retry(total=2, status_forcelist=[418])
+        resp = self.pool.request('GET', '/successful_retry',
+                                 headers=headers, retries=retry)
+        self.assertEqual(resp.status, 200)
+        self.assertEqual(resp.retries.total, 1)
+        # URLFetch use absolute urls.
+        self.assertEqual(resp.retries.history,
+                         [RequestHistory('GET',
+                                         self.pool._absolute_url('/successful_retry'),
+                                         None, 418, None)])
+
     #test_max_retry = None
     #test_disabled_retry = None
+    # URLFetch resolve redirects.
```

Comment:
Can we reword this to be slightly longer and clearer? Maybe

```python
# We don't need these tests because URLFetch resolves its own redirects.
```